qt/widgets/common/src/MantidTreeWidgetItem.cpp

```diff
@@ -65,6 +65,12 @@ bool MantidTreeWidgetItem::operator<(const QTreeWidgetItem &other) const {
       return true;
     return false;
   }
+  // If both should eb sorted and the scheme is set to ByMemorySize ... 
+  else if (m_parent->getSortScheme() == MantidItemSortScheme::ByMemorySize) {
+    if (this->getMemorySize() < mantidOther->getMemorySize())
+      return true;
+    return false;
+  }
```

Comment:
The reason I didn't do that, and I could do that, is because the code around it is older and followed that method of writing it out. I can re-write my part to follow that if you think it's best?

Reply:
Yes I see you wish to retain consistency. Perhaps you could make similar changes elsewhere in the code to make it shorter.

Reply:
Changes made and should be in the next push